plugins/flutter_aepmessaging/lib/src/aepmessaging_message.dart

```diff
@@ -0,0 +1,75 @@
+/// Copyright 2023 Adobe. All rights reserved.
+/// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License. You may obtain a copy
+/// of the License at http://www.apache.org/licenses/LICENSE-2.0
+/// Unless required by applicable law or agreed to in writing, software distributed under
+/// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+/// OF ANY KIND, either express or implied. See the License for the specific language
+/// governing permissions and limitations under the License.
+
+import 'package:flutter/services.dart';
+import 'package:flutter_aepcore/flutter_aepcore.dart';
+import 'package:flutter_aepmessaging/flutter_aepmessaging.dart';
+
+/// Represents a message object retrieved from Adobe Experience Platform
+class Message extends Showable {
+  static const MethodChannel _channel =
+      const MethodChannel('flutter_aepmessaging');
+
+  late String id;
+  late bool autoTrack;
+
+  Message(
```

Comment:
Can we check null or empty for message before all the methods in this file?  Now we are getting error where there is empty message while calling these methods.
[VERBOSE-2:dart_vm_initializer.cc(41)] Unhandled Exception: RangeError (index): Invalid value: Valid value range is empty: 0